CONTRIBUTING.md

```diff
@@ -94,137 +65,217 @@ script before each commit and abort the commit on error. To
+1. Install the latest version of [Visual Studio Code](https://code.visualstudio.com/).
 
-To automatically generate the test coverage report, add the `--coverage` flag to the `test` script defined in [package.json](RetrospectiveExtension.Frontend/package.json). After the test run is completed, coverage statistics will then be reported in the newly created `coverage` directory.
+2. Install the [Docker](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) extension for
+Visual Studio Code.
 
-#### Mocks
+3. Install the
+[Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension for Visual Studio Code.
 
-In order to simulate API calls and some external module functionality, jest mocks were created and utilized. Mocks that
-are expected to used by more than one test file are located in the [mock folder](RetrospectiveExtension.Frontend/components/__mocks__), and organized by module. If mocks are shared by the majority of tests, such as API mocks, add the creation of the mock to the [test setup file](RetrospectiveExtension.Frontend/components/__tests__/setupTests.tsx).
+4. Check out this repository and open the parent folder in Visual Studio Code.
 
-#### Snapshots
+5. Follow the steps outlined in the [Build](#build) section to build, test, and deploy development versions of the extension.
 
-To ensure proper rendering of components, snapshots tests are being used compare expected component rendering state
-against its actual state. Snapshot tests will fail when changes are made to components that are not accounted for
-through updates to these stored snapshots.
+---
 
-To update snapshots, delete the snapshot for the component that you are testing, (located in the [snapshots folder](RetrospectiveExtension.Frontend/components/__tests__/__snapshots__))
-and run the test command. On test run completion, new snapshots should be created. Please check the newly created
-snapshot file, to ensure that the expected changes are present, and include the snapshot in your pull request.
+- The first time this repository is opened in Visual Studio Code, the .devcontainer folder will be detected. In the bottom right-hand corner of the screen, a prompt will be displayed: “Folder contains a Dev Container configuration file. Reopen folder to develop in a container.”.
+- Selecting the “Reopen in Container” option will automatically start the process of creating the Dev Container; this may take a few minutes the first time the container is created, or any time the settings for the container have changed and the container needs to be recreated.
+- If there are issues running the `ci.sh` script in a clean checkout, check the `Files: Eol` setting in Visual Studio Code. Change it to \n, open the `ci.sh` file, save it, and retry the script.
 
-### Test using Hot Reload and Debug
+### Windows Subsystem For Linux
```

Comment:
happy for suggestions on this section - I was able to use the WSL, but I configured my WSL setup partially a few weeks ago and so my instructions may not be 100% accurate.

Reply:
Seems fine to me.

Reply:
Agreed!